### ISSUE TITLE
REQUIRED process [hcplanner_exec-1] has died!

### ISSUE THREAD

**Issue by 1OngJ** [HKUST-Aerial-Robotics/FC-Planner#14]
我跟着文档中的推荐流程去安装相关程序，在我执行完快速开始中roslaunch hierarchical_coverage_planner mbs.launch 
这条语句后，点击2d nav goal标点开始之后，终端出现报错，请问我如何解决这个报错问题。
![微信图片_20240930023403](https://github.com/user-attachments/assets/d53c9fe8-231c-41ed-8bcb-26b00ffc2736)

**Comment by Chen-Albert-FENG** [comment]
Sry for the late reply. I have updated the code and you can try it again when you feel free. :)

**Comment by 1OngJ** [comment]
It doesn't work for me . The same error happen again.

**Comment by 1OngJ** [comment]
Thanks for your work.I have updated and it shows me the same error.Could you tell me why it happen and maybe it is the problem by my platform.

**Comment by Namelessman-Tom** [comment]
I ran into the same error. I have installed all the dependencies and tried to launch the example(two launch files), but rviz displayed nothing but blank. I guess the model did not being loaded into the process or else rviz should show the model(I don't know). When I try to set a nav goal, the process (but it gave the planning time meaning that it actually have done the work?) died.

By the way, the other all launch files in `src/hierarchical_coverage_planner/launch`  throw me the same error. 

Thank you for noticing this problem and give it a repair.
